### PR TITLE
Add coinbase

### PR DESCRIPTION
**Pull request by kocsenc**

### CODE REVIEW

File: chapters/bizleg-case-studies/crowdfunding/coinbase.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+Fairly arbitrary, but Will is a Coinbase user so we went for it.
+
+
+## Coinbase Organizational Details
+
+- Is the subject of your profile a corporate entity?
+> Yes, [Coinbase is a corporation.][1]
+
+- What type?
```
Comment (ryansb) [inline chapters/bizleg-case-studies/crowdfunding/coinbase.md]: You inverted the template, the idea was to do:

> the question? 

Your answer

**Comment by ryansb** [comment]
Please standardize your format.

**Comment by qwertos** [comment]
Looks good. Merged. :+1:

**Comment by ryansb** [comment]
:-1: does not look good. It puts *everything* in one giant block quote. 

> You need a blank line between blocks

And non blocs

**Comment by ryansb** [comment]
@kocsenc can you fix this (and also preview your rendered changes before PR'ing in the future)

**Comment by kocsenc** [comment]
@ryansb so, 
- there are sample markdown files that do not follow the blockquote convention.
- The renderer I was using had no issue rendering the second format, I will use GH flavored to make sure its ok next time

**Comment by kocsenc** [comment]
Making new PR fixing this

**Pull request merged**